solverdummy/requirements.txt

```
@@ -0,0 +1,3 @@
+pyprecice>=0.1
```

```suggestion
pyprecice==2.0
```

Matches all adapter version based preCICE 2.0 with arbitrary patch and adapter version.